utils/api.js

```diff
@@ -70,6 +70,7 @@ export function requestNodeScores(chain, network) {
  * @returns {{fastestFee:number, halfHourFee:number, hourFee:number}} Recommended fees
  */
 export function requestFees(options) {
+  mainLog.info('Fetching fees from bitcoinfees.earn.com')
```

Comment:
Maybe let's use `BASE_URL`  for future proofness? it sometimes happens that a constant get changed and logs remain outdated

Reply:
update